### ISSUE TITLE
Make desktop config work in non-Guix distros

### ISSUE THREAD

**Issue by HumHongeKamyaab** [daviwil/dotfiles#2]
Guix can be installed over top of Ubuntu.
Once you install Guix binary in Ubuntu.
Which commands need to be to so that your entire config can be replicated over my ubuntu. 
I dont want to mess with Guix OS installation now, also I dont want to mess with my perfectly working Ubuntu to try this.

Need to solve this before issue #1

**Comment by daviwil** [comment]
Yep, I think showing how to install on another distro will be helpful since, like you mention, it's possible to use Guix there without using it as your whole distro.
